man/man7/zpool-features.7

```diff
@@ -507,6 +507,15 @@ once either of the limit properties has been set on a dataset
 and will never return to being
 .Sy enabled .
 .
+.feature com.delphix head_errlog no
+This feature enables the upgraded version of errlog, which required an on-disk
+error log format change.
+Now the error log of each head dataset is stored separately in the zap object
+and keyed by the head id.
+With this feature enabled, every dataset affected by an error block is listed
+in the output of
+.Nm zpool Cm status .
```

Comment:
This could also use one of the standard state machine spiels, I think `\*[instant-never]` for this (but dunno)?

Reply:
I didn't understand your comment, could you elaborate?

Reply:
All (most? but I tried to make it all) other features have a short paragraph at the end describing the disabled<->enabled<->active state machine – for example `embedded_data` has `\*[instant-never]` (which expands to `.No This feature becomes Sy active No as soon as it is enabled and will never return to being Sy enabled .`), checksum features use `.checksum-spiel SUM` (which expands to `.No When the .Sy SUM No feature is set to Sy enabled , No the administrator can turn on the Sy SUM No checksum on any ...` &c.), `large_blocks` has a different set of conditions (`No This feature becomes Sy active No once a dataset contains a file with a block size larger than 128kB, and will return to being Sy enabled No once all ...`).

If this behaves as I think it does – i.e. like `embedded_data`, in that it activates instantly and you can never turn it off – then all you need to do here is add
```
.Nm zpool Cm status .
.Pp
\*[instant-never]
.
.feature ...
```
Otherwise copy-and-modify one of the other descriptions.

Reply:
Thank you, fixed in the upcoming rebase.